llvm/lib/Target/X86/X86FixupSetCC.cpp

```diff
@@ -98,9 +106,8 @@ bool X86FixupSetCCPass::runOnMachineFunction(MachineFunction &MF) {
         continue;
 
       // On 32-bit, we need to be careful to force an ABCD register.
-      const TargetRegisterClass *RC = MF.getSubtarget<X86Subtarget>().is64Bit()
-                                          ? &X86::GR32RegClass
-                                          : &X86::GR32_ABCDRegClass;
+      const TargetRegisterClass *RC =
```

Comment:
Seems unrelated.

Reply:
It's kind of related.  There are 2 uses of `X86Subtarget` after this PR, so I define the variable `ST` for it.  Unless we'd like to define the variable even for 1 use,  I can not extract this into a separate NFC change.